examples/booking_app/lib/feature_hotel_favorites/views/hotel_favorites_page.dart

```diff
@@ -27,11 +27,11 @@ class FavoritesPage extends StatelessWidget {
               ),
             ),
           ),
-          RxResultBuilder<FavoriteHotelsBlocType, List<Hotel>>(
+          RxResultBuilder<HotelFavoritesBlocType, List<Hotel>>(
             state: (bloc) => bloc.states.favoriteHotels,
             buildLoading: (ctx, bloc) => LoadingWidget(),
             buildError: (ctx, error, bloc) => ErrorRetryWidget(
-              onReloadTap: () => RxBlocProvider.of<FavoriteHotelsBlocType>(ctx)
+              onReloadTap: () => RxBlocProvider.of<HotelFavoritesBlocType>(ctx)
```

Comment:
This is just a style thing but everywhere in the same file we use `context` instead of the `ctx` abbreviation.
It was probably autogenerated but I prefer to avoid abbreviations for names.

Reply:
Done